stl/inc/vector

```diff
@@ -1891,14 +1947,14 @@ public:
         return this->_Myptr;
     }
 
-    friend void swap(_Vb_reference _Left, _Vb_reference _Right) noexcept {
+    _CONSTEXPR20_DYNALLOC friend void swap(_Vb_reference _Left, _Vb_reference _Right) noexcept {

```

Comment:
We conventionally say `friend` before `constexpr` (or macros for `constexpr`). Occurs elsewhere in this PR.

Reply:
Thanks, fixed